### PR TITLE
Fix opacity bug on Android (#4)

### PR DESCRIPTION
**Pull request by lucasbento**

### CODE REVIEW

**Comment by alexmngn** [comment]
I don't have any issue running the current version on Android (Nexus 6P Emulator + API 25) and don't understand the reason of this PR. Can you explain what issue do you have? Which Android version?

See attached screenshot:

screen shot 2017-01-19 at 11 43 16

**Comment by lucasbento** [comment]
Yes, it indeed works fine on react-native `0.31`, this is how it looks on `0.39.2` on a physical device:

screenshot_20170126-110538

Notice that the image parts that are supposed to have the opacity don't appear.

**Comment by alexmngn** [comment]
This pull request doesn't make it looks like my screenshot anymore. You have a light background whereas I have a dark background. Can you make the change so it looks exactly the same?

**Comment by lucasbento** [comment]
@alexmngn: oh, okay, since I used a white background I assumed it would be always white, gonna fix that.

**Comment by sibelius** [comment]
@alexmngn @lucasbento can we get this in?

**Comment by alexmngn** [comment]
@sibelius The pull request still doesn't look good to me. Waiting for @lucasbento to make the changes.

**Comment by lucasbento** [comment]
@alexmngn: would it be something like this?

screenshot_20170201-101335

**Comment by alexmngn** [comment]
@lucasbento Yes that's supposed to be like this.

**Comment by lucasbento** [comment]
@alexmngn: good, just updated it, would you mind checking it again?

File: index.js [hunk 1]
```diff
@@ -342,9 +348,20 @@ class ImageCrop extends Component {
...
 						}]}
-					/>
+					>
+						{Platform.OS === 'android' && isLower && (
+              <View style={{
+								flex: 1,
+								backgroundColor: (isLower) ? 'rgba(0, 0, 0, 0.5)' : 'transparent',
+							}} />
+						)}
+					</Image>
```
Comment (alexmngn) [inline index.js:364]: Indentation not correct on this file.

**Review by alexmngn — CHANGES_REQUESTED** [review]

**Comment by sibelius** [comment]
@alexmngn could you please fix the missing styles errors?

it has been more than 2 months to merge this simple PR

**Comment by sibelius** [comment]
@alexmngn consider using prettier to avoid this in the future https://medium.com/@okonetchnikov/make-linting-great-again-f3890e1ad6b8#.d9w1oimtl

**Comment by alexmngn** [comment]
@sibelius The project has a .eslintrc file and has a strict rule, you should make sure your editor is setup to accept those rules.

```
    "rules": {
        "indent": [2, "tab", {"SwitchCase": 1}],
```

I will look into it very soon and might add a few features but I don't have much time at the moment. Sorry about that.

**Comment by sibelius** [comment]
> a strict rule 

how is this making us more productive?

I have no problem with spaces vs tabs, but I just would like to get our changes in the main repo to help improve the package for everybody.

but I think you are more concern with making lint work before making more features available

**Pull request closed without merging**